package.json

```
@@ -10,6 +10,8 @@
   "scripts": {
     "ci:version": "pnpm exec changeset version",
     "ci:publish": "pnpm changeset publish",
+    "build:all": "pnpm wasm:build && pnpm ts:build",
+    "ts:build": "pnpm --dir ./packages/pglite build",
```

We can use the `pnpm -r` to build all packages, I've added a filter to only do packages (React, Vue, Sync etc). This will exclude the `/docs` and `/demos` (we we add them)

```suggestion
    "ts:build": "pnpm -r --filter "./packages/**" build",
```